objects/src/notes/file.rs

```diff
@@ -0,0 +1,64 @@
+use vm_core::utils::{ByteReader, ByteWriter, Deserializable, Serializable};
+use vm_processor::DeserializationError;
+
+use super::{Note, NoteDetails, NoteId, NoteInclusionProof};
+
+// NOTE FILE
+// ================================================================================================
+
+/// A serialized representation of a note.
+pub enum NoteFile {
```

Comment:
Let's add implementations for `From<NoteId>` and `From<NoteDetails>` for `NoteFile`.

Reply:
Done.